build/teamcity/cockroach/nightlies/roachtest_compile_bits.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-source "$(dirname $0)/roachtest_arch_util.sh"
+source $root/build/teamcity/util/roachtest_arch_util.sh
```

Comment:
As we depend on the variable `$root`, this won't work unless the person running the script has already executed `teamcity-support.sh` in this shell. (And then e.g. ran this script with `. roachtest_compile_bits.sh`) Maybe true in all situations where we use this script, but worth pointing out.

Reply:
Right. It'd be nice to export `root` or something similar. I find the other pattern (i.e., `dirname`) to be error-prone, especially when refactoring.

Reply:
I would add a comment like ("must run `build/teamcity-support.sh` before executing this script") at the top here just to make the dependency clear, then.